### DISCUSSION TITLE
Stuck at last step, Please help!

### DISCUSSION THREAD

**Discussion by drashish87** [keatontaylor/alexa-actions#197] [Q&A]
Hi guys, I would appreciate if someone can help me. I have followed the guide very closely. I am stuck at the last step. I cant hear any voice when running the test script
service: script.activate_alexa_actionable_notification
data:
  text: Yes or no?
  event_id: alexa_actionable_notification_test
  alexa_device: media_player.<Your Media Player>

Also, i cant see any response under events > listening 

However, interestingly, when i run the command from Alexa developer skill page>test?open actionable notifications, events>is listening to this event.

Can you suggest whats going wrong? I have spent hours but cant figure this out.

**ALSO there is an error in the guide, the actionable script needs to be copied in scripts.yaml not configurations.yaml as mentioned. Might be helpful to correct it for other!**

FAQ's
If you hear nothing or you don't receive the expected response, check your setup.

Is the skill listed within your Alexa App on your Smartphone? **YES**
Is the volume up? **YES**
Did you connect your device and not to a device of your wife's or husband's account?**NO**
Did you provide the right ID of your skill? **YES**
In the developer console check the Analytics of your skill. (Utterances count up with some latency.) **Only goes up when i run the command from Alexa developer testing not from HA**

<img width="1696" alt="Screenshot 2023-07-21 at 19 45 41" src="https://github.com/keatontaylor/alexa-actions/assets/136771698/ee4f4159-a4b2-48f1-85ad-af3c287a1124">
<img width="1696" alt="Screenshot 2023-07-21 at 19 46 35" src="https://github.com/keatontaylor/alexa-actions/assets/136771698/a889fc59-a42d-4c90-91cf-6b8a80b64c98">

**Comment by DEADSEC-SECURITY** [comment]
Can you put quotes around the texts when running the script?

**Reply by DEADSEC-SECURITY** [reply]
`text: Yes or No `should be `text: "Yes or No"` for example.

**Reply by DEADSEC-SECURITY** [reply]
Do the same for the event_id and let me know if it still gives the same issue.

**Comment by DEADSEC-SECURITY** [comment]
Another thing you can try is to make your alexa device talk using the developer tools notification service for alexa devices
![image](https://github.com/keatontaylor/alexa-actions/assets/45077772/b947bc01-b780-420a-b324-1f09a9ef2410)

If the device say anything when running via there, could mean you have a bad setup on ha side.

**Comment by DEADSEC-SECURITY** [comment]
Can you try triggering alexa with out the script but with the same logic.
Put some test data into the helper variable, then trigger the service to start the skill via the dev tools. If that works, then it could be an issue with the script it self.

**Comment by drashish87** [comment]
Thanks a lot for replying back it is much appreciated. Yes, I tried few of the above suggestions already but it dint work. I kept wondering because I am in the UK and the skill was being made in English-US was causing the issue? maybe. I came across a YouTube video with a powershell script to do the same, that worked and I have access to Alexa. Many thanks for developing the skill.

**Reply by Srpbtw1** [reply]
Can i have the link. im very fustraited rn lmao

**Reply by drashish87** [reply]
You'll be up and running in 5 mins with this. https://www.youtube.com/watch?v=RlnfmW9MU94&t=522s
Remember I had a lot of issues in starting the powershell script to install it was due to windows permissions issue just google unable to run powershell script and you will be presented with couple of commands that help run this kind of script and you'll be good to go.

**Reply by Srpbtw1** [reply]
Thank you so much!

**Reply by Srpbtw1** [reply]
Its not working for me, ive been trying for an hour, i get all the powershell scripts working, but when i try to like get alexa to actually send a notification its not working.

**Reply by drashish87** [reply]
Forward the error you are getting and I will try my best to help. For me Alexa action notifications are working perfectly!
